paddle/operators/activation_op.cc

```diff
@@ -166,6 +166,23 @@ class SoftReluOpMaker : public framework::OpProtoAndCheckerMaker {
 };
 
 template <typename AttrType>
+class ELUOpMaker : public framework::OpProtoAndCheckerMaker {
+ public:
+  ELUOpMaker(framework::OpProto *proto, framework::OpAttrChecker *op_checker)
+      : OpProtoAndCheckerMaker(proto, op_checker) {
+    AddInput("X", "Input of ELU operator");
+    AddOutput("Y", "Output of ELU operator");
```

Comment:
it could be change to `AddOutput("Y", "Output of ELU operator").NotInGradient();`, cause y is not used in `ELUGradKernel`.

Reply:
@tonyyang-svail Thanks for pointing that out, you are right.

However, after reexamining the math, I find there is a better way to rewrite the gradient formulation which does require using `y`. I'm working on it, and this PR will be updated soon.

Reply:
Done. 
Now the gradient of negative part is computed as follow:
'''
dy * (y + alpha) * (x < static_cast<T>(0)).template cast<T>();
'''
So now value of `y` is used in the gradient calculation.